src/Driver/SolverTypes/ImplicitSolverType.jl

```diff
@@ -0,0 +1,69 @@
+
+export ImplicitSolverType
+
+"""
+# Description
+    ImplicitSolverType(;
+        solver_method = KenCarp4,
+    )
+
+This solver type constructs an ODE solver using an implicit
+method.
+
+# Arguments
+- `solver_method` (Function): Function defining the implicit
+    solver.
+    Default: `KenCarp4`
+"""
+struct ImplicitSolverType <: AbstractSolverType
+    # Function for an implicit method
+    solver_method::Function
+
+    function ImplicitSolverType(
+        alg;
+        solver_method = ODESolvers.DiffEqJLConstructor(alg),
```

Comment:
I think this answers my initial question: `alg` contains the timestepping info, and I'm assuming this also somehow attaches information about the actual implicit solver algorithm?